metashare/api/models.py

```diff
@@ -213,11 +213,42 @@ class Task(mixins.HashIdMixin, mixins.TimestampsMixin, SlugMixin, models.Model):
 
     slug_class = TaskSlug
 
+    def __str__(self):
+        return self.name
+
     class Meta:
         ordering = ("-created_at", "name")
         unique_together = (("name", "project"),)
 
 
+SCRATCH_ORG_TYPES = Choices("Dev", "QA")
```

Comment:
@wlonk Should these (and also the `ORG_TYPES` choices) be translatable on the back-end?

Reply:
That is, I'm trying to decide whether these values should be directly displayed on the front end (in which case they should probably be translatable on the back end), or should be hardcoded as display types for the front end (and translatable on the front end).

Reply:
I'm not sure; I could go either way. If I imagine this API being used by other frontends than ours, I'd rather it _not_ translate these strings, but that may be a fantasy anyway, so I'm not sure.

Reply:
Yeah, that makes sense to me. I tend to think the front end should be in control of display values, so I'll follow that here and make then translatable on the front end. I think it just threw me off that these were capitalized -- I expected `dev` and `qa` instead of `Dev` and `QA` -- but I'm not sure why.